modules/@apostrophecms/notification/ui/apos/components/AposNotification.vue

```diff
@@ -118,6 +118,7 @@ export default {
     align-items: center;
     min-width: 200px;
     max-width: 400px;
+    margin-top: 8px;
```

Comment:
I think they only need top margin when they're the sibling of another one. I'm imagining a nested style:
```
& + .apos-notification {
   margin-top: 8px;
}
```